man/systemd.network.xml

```diff
@@ -706,6 +706,12 @@
             <citerefentry project='man-pages'><refentrytitle>inet_pton</refentrytitle><manvolnum>3</manvolnum></citerefentry>.</para>
           </listitem>
         </varlistentry>
+        <varlistentry>
+          <term><varname>OnDevice=</varname></term>
+          <listitem>
+            <para>A boolean. Specifies whether to take device as default route.</para>
```

Comment:
By setting this all the packets will be sent via this interface. In case of PPP  it will reach to the other end . From there it would be routed to according to the rules there .